### PR TITLE
Localdev fork mainnet

### PR DESCRIPTION
**Pull request by 0xTARC**
## Testing Instructions
Follow the new instructions in the README for `Running with Localhost Network backend`

Ensure you can open a position via underlier in the UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,46 @@
...
+    yarn transferLocalhostTokens --to=<first_account_address>
+    ```
+    to transfer stablecoins into your imported account. After running, you should have thousands of stablecoins to test with!
+
+1. In another terminal, run the frontend with
+    ```sh
+    yarn dev
+    ```
+
+1. You're done! You can now open positions via underlier and buidl!
```
Comment (Rudi114) [inline README.md:44]: Cringy

File: src/constants/chains.ts [hunk 2]
```diff
@@ -36,6 +38,20 @@ export type ChainConfig = {
 export const INITIAL_APP_CHAIN_ID = Number(DEFAULT_CHAIN_ID) as ChainsValues
 
 export const chainsConfig: Record<ChainsValues, ChainConfig> = {
+  [Chains.localhost]: {
+    id: Chains.mainnet,
+    name: 'Localhost Mainnet',
+    shortName: 'Localhost Mainnet',
+    chainId: Chains.localhost,
+    chainIdHex: '0x539',
```
Comment (Rudi114) [inline src/constants/chains.ts:46]: Where does 539 come from?

File: README.md [hunk 3]
```diff
@@ -2,10 +2,46 @@
...
+1. Import the first ganache account into your wallet via private key. See here for metamask [instructions](https://metamask.zendesk.com/hc/en-us/articles/360015489331-How-to-import-an-account#h_01G01W07NV7Q94M7P1EBD5BYM4).
+
+1. Ensure you have the Localhost Network in your wallet.
+
+1. Copy the address of the first ganache account and run
+    ```sh
+    yarn transferLocalhostTokens --to=<first_account_address>
+    ```
+    to transfer stablecoins into your imported account. After running, you should have thousands of stablecoins to test with!
+
```
Comment (Rudi114) [inline README.md:38]: Might be worth adding something about importing tokens to local network account after transfering them to your wallet. (Metamask link and contract addresses)

File: README.md [hunk 4]
```diff
@@ -2,10 +2,46 @@
...
-1. Clone the project: `git clone git@github.com:fiatdao/fiat-ui.git`.
-2. Create a .env.local file: `cp .env.example .env.local`.
-3. Install Node Dependencies: `yarn install`.
-4. Start the app with `yarn dev`.
+1. Clone the project:
+    ```sh
+    git clone git@github.com:fiatdao/fiat-ui.git
+    ```
+
+1. Create a `.env.local` file:
```
[the remark this answers is not in the document]
Comment (0xTARC) [reply README.md:10]: markdown hack
in a rendered readme, the
```
1.
1.
1.
```
turns into
```
1.
2.
3.
```

so it makes it much easier to add and remove steps in an ordered list. if do the other way with 1,2,3, and add a step in the middle you'd to update the number of all the other steps

File: README.md (hunk 1 above)
Comment (0xTARC) [reply README.md:44]: are you not a buidler

File: src/constants/chains.ts (hunk 2 above)
Comment (0xTARC) [reply src/constants/chains.ts:46]: hex translation of 1337. 1337 is the default ganache chainId

File: README.md (hunk 3 above)
Comment (0xTARC) [reply README.md:39]: good idea, addressed in #98ad880

**Pull request merged**